Anime 등록, 수정 문서화 #17

jaycobcoder opened the pull request:
## 📝 개요
docs 일부 수정, 애니 추가, 수정 api 문서화

## 🚀 변경사항
1. 단일 객체 응답 시 바로 응답하도록 변경
2. 1에 따른 문서화 수정
3. 애니 추가, 수정 api 문서화
4. 어드민 도메인 추가

## 🔗 관련 이슈
#17 

## ➕ 기타

![제목 없음](https://github.com/oduck-team/oduck-server/assets/113520315/d66f1268-8af9-4ace-8d43-a33cc22a7a63)
어제 수정 사항 커밋들 `feature/17` 브랜치에 push했습니다. `index.adoc` 파일만 push하고 있지 않았는데, 하니님 마지된 거 확인 이후 `develope` 브랜치 pull 한 다음에 adoc 파일을 push했는데 문제 있을까요?😱

yunhozz reviewed (approved):
🚀

jaycobcoder committed:
refactor: Anime Controller 응답 변경 #17

Dto를 SingleResponse에 감싸서 return하는 것이 아닌, 바로 return하도록 변경하였습니다. animeId를 id로 변경하였습니다. Dto의 Broadcast 클래스를 삭제하였습니다.

jaycobcoder committed:
test: Anime Controller 응답 변경에 따른 테스트 변경 #17

jaycobcoder committed:
feat: Admin 도메인 추가, 애니 등록, 수정 관련 추가 #17

jaycobcoder committed:
test: 애니 등록, 수정 관련 테스트 #17

jaycobcoder committed:
docs: index.adoc의 admin 도메인 추가 #17

hanyMK commented:
![image](https://github.com/oduck-team/oduck-server/assets/119990091/8abd1a27-2e07-4af2-b7a1-ef3db32a065a)
![image](https://github.com/oduck-team/oduck-server/assets/119990091/c9734ce5-144a-41cd-aba0-596220e81c93)
1번 이미지 와 2번 이미지 중간에 .adoc를 작성해서 충돌이 일어난것 같습니다.

jaycobcoder committed:
refactor: anime 엔티티 변경 #17

year 컬럼을 release_year로 변경하였고, 엔티티에서는 year로 사용하도록 변경하였습니다. summary 글자수를 255에서 600으로 늘렸습니다.

FaberJoo commented:
pull 땡기신 다음 충돌 해결하시고 push 하시면 되십니다!

jaycobcoder committed:
Merge branch 'develop' into feature/17
